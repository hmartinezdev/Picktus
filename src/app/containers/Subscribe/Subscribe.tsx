import SubscribeHandler from '@components/SubscribeHandler';
import React, { Component } from 'react';

class Subscribe extends Component {
  public render(): React.ReactElement<Subscribe> {
    return <SubscribeHandler />;
  }
}

export default Subscribe;
