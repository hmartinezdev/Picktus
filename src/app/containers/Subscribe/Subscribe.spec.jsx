import { shallow } from 'enzyme';
import React from 'react';
import Subscribe from './Subscribe';

let wrapper;

const setup = () =>
  shallow(
    <Subscribe>
      <div />
    </Subscribe>
  );

describe('<Subscribe />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
