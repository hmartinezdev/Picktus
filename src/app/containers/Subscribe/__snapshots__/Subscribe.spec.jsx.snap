// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Subscribe /> should render properly 1`] = `
<div
  className="container container--undefined"
>
  <div
    className="logo logo--undefined"
  >
    <Logo />
  </div>
  <div
    className="form"
  >
    <Connect(SubscribeHandler) />
  </div>
  <style
    jsx={true}
  >
    
          .container {
            width: 100%;
            height: 100vh;
            display: flex;
            align-items: center;
            flex-direction: column;
            padding: 2rem 0;
            box-sizing: border-box;
            overflow: hidden;
            opacity: 1;
            transition: opacity 300ms ease-out;
          }

          .container--exiting,
          .container--exited {
            position: absolute;
            opacity: 0;
          }

          .logo {
            background-color: #2d3047;
            box-shadow: 11px 10px 34px -7px rgba(0, 0, 0, 0.49);
            display: flex;
            position: relative;
            border-radius: 3px;
          }

          .logo--exiting,
          .logo--exited {
            box-shadow: none;
          }

          .form {
            height: 100%;
            width: 100%;
            display: flex;
            flex-direction: column;
            justify-content: flex-start;
            align-items: center;
            margin-top: 2.5rem;
            opacity: 1;
            transition: opacity 130ms ease-out;
          }
        
  </style>
</div>
`;
